test/command_callback/test_glslls_command_callback.vader

```diff
@@ -0,0 +1,37 @@
+Before:
+  Save g:ale_glsl_glslls_executable
+  Save g:ale_glsl_glslls_logfile
+
+  unlet! g:ale_glsl_glslls_executable
+  unlet! g:ale_glsl_glslls_logfile
+
+  runtime ale_linters/glsl/glslls.vim
+  call ale#test#SetDirectory('/testplugin/test/command_callback')
+
+After:
+  Restore
+  call ale#linter#Reset()
+
+Execute(Executable should default to 'glslls'):
+  AssertEqual
+  \ 'glslls',
+  \ ale_linters#glsl#glslls#GetExecutable(bufnr(''))
+
+Execute(Executable should be configurable):
+  let g:ale_glsl_glslls_executable = 'foobar'
+  AssertEqual
+  \ 'foobar',
+  \ ale_linters#glsl#glslls#GetExecutable(bufnr(''))
+
+Execute(Command should use executable):
+  let command1 = ale_linters#glsl#glslls#GetCommand(bufnr(''))
+  AssertEqual command1, "'glslls' --stdin"
```

Comment:
Use `ale#Escape` on these test values instead. Escaping works differently on Windows, and the tests are failing in AppVeyor. Prefer double quoted strings. You can escape single quotes in single quoted strings by doubling them, but you won't need to do that here.